docs/modules/i3s/recipes/building-scene-layer.md

```diff
@@ -1,95 +1,96 @@
-```
-
-The loader returns flattened array `sublayers` that contain only `3DObject` layers from BSL sublayers tree.
-
-### Show sublayers in deck.gl
-
-```jsx
-  import { StaticMap } from "react-map-gl";
-  import DeckGL from "@deck.gl/react";
-  import { Tile3DLayer } from "@deck.gl/geo-layers";
-  import { COORDINATE_SYSTEM } from "@deck.gl/core";
-  import { I3SLoader } from "@loaders.gl/i3s";
-
-  function renderLayer() {
-    const tile3dLayers = sublayers.map((layer) => {
-      if (!layer.visibility) {
-        return null;
-      }
-      const loadOptions = {
-        i3s: {coordinateSystem: COORDINATE_SYSTEM.LNGLAT_OFFSETS},
-      };
-      if (token) {
-        loadOptions.i3s.token = token;
-      }
-      return new Tile3DLayer({
-        id: `tile3d-layer-${layer.id}`,
-        data: layer.url,
-        loader: I3SLoader,
-        loadOptions,
-      });
-    });
-  }
-
-  <DeckGL
-    id={id}
-    layers={renderLayers()}
-    viewState={viewState}
-    onViewStateChange={onViewStateChangeHandler}
-  >
-    <StaticMap reuseMaps mapStyle={mapStyle} />
-  </DeckGL>
-```
-
-Take a look on sublayer properties:
-* `id` - unique identifier required to set deck.gl layer id;
-* `url` - `3DObject` layer url;
-* `visibility` - BSL has initial visibility state for all sublayers. Use this property to controll sublayers visibility.
-
-### Control sublayers visibility
-
-`I3SBuildingSceneLayerLoader` returns also `header` object. `header.sublayers` is a JSON nested tree that can be used to visualize BSL explorer panel.
-
-![BSL Explorer visualization](bsl-explorer.png)
-
-With such a panel it is possible to change a sublayer visibility property:
-
-```javascript
-  function onChangeHandler(treeItem) => {
-    const sublayer = sublayers.find((sublayer) => sublayer.id === treeItem.id);
-    if (sublayer) {
-      sublayer.visibility = !sublayer.visibility;
-    }
-  }
-```
\ No newline at end of file
+# I3S Building Scene Layer (BSL)
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v3.2-blue.svg?style=flat-square" alt="From-v3.2" />
+</p>
+
+A building scene layer can represent 3D model content based on BIM structuring disciplines such as architectural or structural, and categories such as windows or walls. In I3S [specification](https://github.com/Esri/i3s-spec) BSL is a layer type.
+
+BSL doesn't have content resources. It is a composite layer that consists of multiple sublayers. Every sublayer can be: `group`, `3DObject` or `Point`.
+
+`group` sublayer is also just a composite layer. It contains further levels of sublayers and provides a nested structure for BSL. As a result, BSL has a tree-like sublayers structure with `3DObject` and `Point` layers;
+
+`3DObject` and `Point` sublayers are content layers. Those layer types are separated types of layers in I3S spec and can be used independently. Loaders.gl supports `3DObject` layer types and doesn't support `Point`.

```

Comment:
These types could perhaps be put in a markdowntable to increase readability?

Here again, more information about these 3DObject layers and what is in them would feel appropriate before we jump into the special case of visualizing them with deck.gl